ssz/utils.py

```diff
@@ -0,0 +1,23 @@
+from ssz.sedes import (
+    boolean,
+)
+
+
+def is_sedes(obj):
+    """
+    Check if `obj` is a sedes object.
+    A sedes object is characterized by having the methods
+    `serialize(obj)` and `deserialize(serial)`.
+    """
+    return hasattr(obj, 'serialize') and hasattr(obj, 'deserialize')
```

Comment:
What do you all think about implementing and dectecting a `sedes` using a common base class (this approach bothers me because it is imprecise).

```python
class BaseSedes(ABC):
    @abstractmethod
    def serialize(self, obj):
        pass

    @abstractmethod
    def deserialize(self, serialized_obj):
        pass
```

This way we can just do an `isinstance` check instead of `hasattr` checks.  Thoughts?